### ISSUE TITLE
Parameter identification regarding B/P value.

### ISSUE THREAD

**Issue by regulus92** [Open-Systems-Pharmacology/PK-Sim#1134]
Hello, everyone. I find in PKSim simulation that B/P values are used seperately in liver plasma clearance calculation and partition coefficients. When B/P value as a partition coefficient is adjusted during parameter identification, it won't be changed in liver plasma clearance calculation correspondingly. Thus, I doubt that in this occasion accuracy of parameter identification is not very reliable. Or anything I don't know?

**Comment by regulus92** [comment]
Same question for logP that is also used in liver plasma clearance calculation.

**Comment by msevestre** [comment]
@regulus92
Those parameters are only used at the time of process creation to calculate intrinsic clearance (and specific clearance) . After that the link is broken. You can see it as a calculator so to speak
The same goes with all other parmeters used in the calculation. For example volume liver is the average liver human volume and may have bothuto do with the individual you are using in your simulation. 

So in short, this parameters are just here to give you an estimate of the resulting specific clearance. This is the parameter that you should optimize along with logP.

We had discussion about removing this feature altogether and only showing the spec clearance in the simulation. This may be implemented that way eventually.

So not a bug. This is by design but I understand that it might be confusing.

**Comment by regulus92** [comment]
@msevestre I get it. Thanks a lot.

**Comment by msevestre** [comment]
Cool. Closing
